src/YieldSpaceAMO.sol

```diff
@@ -215,22 +215,23 @@ contract YieldSpaceAMO is Owned {
 
     /// @notice remove a new series in the AMO, to keep gas costs in place
     /// @param seriesId the series being removed
-    /// @param seriesIndex the index in the seriesIterator for the series being removed
-    function removeSeries(bytes6 seriesId, uint256 seriesIndex) public onlyByOwnGov {
-        require (seriesId == seriesIterator[seriesIndex], "Index mismatch");
+    function removeSeries(bytes6 seriesId) public onlyByOwnGov {
```

Comment:
i am v confused, i dont know why this change is here, i didnt make it.  could there have been differences in the upstream when i was on the other fork?  all other changes in this pr look good